### ISSUE TITLE
Subspec issue: None of your spec sources contain a spec satisfying the dependency

### ISSUE THREAD

**Issue by iDevelopper** [CocoaPods/CocoaPods#6970]
* [X] I've read and understood the [*CONTRIBUTING guidelines and have done my best effort to follow](https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md).

# Report

## What did you do?

Run `pod install` in a new project.

## What did you expect to happen?

Install all pod dependencies correctly. Subspec not found

## What happened instead?

[!] Unable to satisfy the following requirements:

- `PBRevealViewController/Swift` required by `Podfile`

None of your spec sources contain a spec satisfying the dependency: `PBRevealViewController/Swift`.

[PBRevealViewController.podspec.zip](https://github.com/CocoaPods/CocoaPods/files/1237837/PBRevealViewController.podspec.zip)

### Stack

```
   CocoaPods : 1.2.1
        Ruby : ruby 2.2.2p95 (2015-04-13 revision 50295) [x86_64-darwin16]
    RubyGems : 2.4.5
        Host : Mac OS X 10.12.6 (16G29)
       Xcode : 8.3.3 (8E3004b)
         Git : git version 2.4.5
Ruby lib dir : /Users/patrickbodet/.rbenv/versions/2.2.2/lib
Repositories : master - https://github.com/CocoaPods/Specs.git @ 279fb3c80141d59ad4a8646ada0ad11decedc9f0
```

### Installation Source

```
Executable Path: /Users/patrickbodet/.rbenv/versions/2.2.2/bin/pod
```

### Plugins

```
cocoapods-deintegrate : 1.0.1
cocoapods-plugins     : 1.0.0
cocoapods-search      : 1.0.0
cocoapods-stats       : 1.0.0
cocoapods-trunk       : 1.2.0
cocoapods-try         : 1.1.0
```

### Podfile

```ruby
# Uncomment the next line to define a global platform for your project
platform :ios, '9.0'

target 'PBRevealSwift' do
    use_frameworks!
    pod 'PBRevealViewController/Swift'

end
```

## Project that demonstrates the issue

[PBRevealSwift.zip](https://github.com/CocoaPods/CocoaPods/files/1237844/PBRevealSwift.zip)

**Comment by orta** [comment]
Hey there, this looks like a great example of a question that you should ask on [StackOverflow](http://stackoverflow.com) - there is an active tag [CocoaPods](http://stackoverflow.com/tags/cocoapods) where people can help out with questions like this. We try to keep the CocoaPods issues focused only on bug reports for the tool, and for upcoming feature requests. Which this doesn't look like.

**Comment by iDevelopper** [comment]
Sorry but why?

Am I missing something?

If you know the answer to my problem, can you put me on the way, please?

Regards,

**Comment by iDevelopper** [comment]
I think you have it in your folder, but it is not referenced in the project. Can you see it within Xcode?

 

Patrick BODET

 

De : Orta <notifications@github.com>
Répondre à : CocoaPods/CocoaPods <reply@reply.github.com>
Date : lundi 21 août 2017 à 13:10
À : CocoaPods/CocoaPods <CocoaPods@noreply.github.com>
Cc : Courriel BODET <patrick.bodet4@wanadoo.fr>, Author <author@noreply.github.com>
Objet : Re: [CocoaPods/CocoaPods] Subspec issue: None of your spec sources contain a spec satisfying the dependency (#6970)

 

Closed #6970.

**Comment by iDevelopper** [comment]
It was affectively a bug. I had tu run pod outdated manually for it to work.
